neuron_reduce/subtree_reductor_func.py

```diff
@@ -30,24 +29,19 @@
 import numpy as np
 import neuron
 from neuron import h
+from neuron import gui
```

Comment:
where did this come from?

Reply:
it was `from neuron import h, gui` at the start, I removed it, and it was working, but then I found out that `import gui` triggers loading of hoc files needed for some sort of string handling, which was necessary for loading other templates